lms/resources/_js_config.py

```diff
@@ -33,7 +33,7 @@ def _consumer_key(self):
     def _ai_getter(self):
         return self._request.find_service(name="ai_getter")
 
-    def add_canvas_file_id(self, canvas_file_id):
+    def add_canvas_file_id(self, course_id, canvas_file_id):
```

Comment:
Added the `course_id` here because it's now needed to generate the URL for the `via_url` API call below